src/app/components/team/UnmatchedMedia.js

```diff
@@ -6,12 +6,16 @@ import Search from '../search/Search';
 import UnmatchedIcon from '../../icons/unmatched.svg';
 
 const UnmatchedMedia = ({ routeParams }) => {
-  const query = {
-    unmatched: [1],
+  const defaultQuery = {
+    unmatched: ['1'],
```

Comment:
Review this change :)

Reply:
![image](https://github.com/meedan/check-web/assets/766082/4649e5c6-7f84-448b-a4ac-21598e41466d)

This was showing in the stateQuery as a string anyway. I edited here to match so the logic for the display of save|reset|apply buttons would work as expected

Reply:
The image above is the behavior when simply opening the Unmatched list, with the change reverted. Reset and apply should not appear.